com.unity.ml-agents/Runtime/Policies/BehaviorParameters.cs

```diff
@@ -150,6 +150,11 @@ public string BehaviorName
         [Tooltip("Use all Sensor components attached to child GameObjects of this Agent.")]
         bool m_UseChildSensors = true;
 
+        [HideInInspector]
+        [SerializeField]
+        [Tooltip("Use all Actuator components attached to child GameObjects of this Agent.")]
+        bool m_UseChildActuators = true;
```

Comment:
We may want to attach Actuators to specific child GameObjects to better organize actions.